kuma/javascript/src/error-boundaries.jsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 
+import GAProvider from './ga-provider.jsx';
 import { gettext } from './l10n.js';
 
 export class AppErrorBoundary extends React.Component {
     state = { error: null };
 
+    static contextType = GAProvider.context;
+
     static getDerivedStateFromError(error) {
         return { error };
     }
 
-    logError() {
-        // https://github.com/mozilla/kuma/issues/5833
-        // When we start to use Sentry, change the method signature to
-        // logError(error, errorInfo) {
+    logError(boundaryName) {
+        // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#event_fields
+        this.context('send', {
```

Comment:
This `this.context` confuses me as well. It must be that your `contextType` definition above tells React to call `useContext` on that object in order to return the value of `this.context`?

Reply:
All these years and I'm still unsure how to use contexts. Because I almost never like them and very rarely think they're needed. I *think* this is how you use it.

Reply:
Before the API I didn't like them either. I.e. I also think the `contextType` thing is not great and not very explicit. But simultaneously it's the right choice here (until we get error boundary hooks).